### DISCUSSION TITLE
[Portico Plugin] Replace the local JATS serialization by the JATS Template Plugin

### DISCUSSION THREAD

**Discussion by jonasraoni** [pkp/pkp-lib#7974] [Proposals]
In order to decrease the maintenance job we can replace the current serialization by the [JATS Template Plugin](https://github.com/pkp/jatsTemplate/).

**Comment by jonasraoni** [comment]
1. Let me know if I should open an issue in the JATS Template Plugin to make it more configurable, like being able to specify a preferred JATS version.

2. As the JATS plugin will become a dependency, it came to my mind that we could use composer to install/manage the OJS plugins, so we could deploy everything as packages and get some stats for free. If you think it's worth I can open an issue to discuss it...

**Comment by NateWr** [comment]
I think we should bring this into the core application, rather than keeping it as a plugin. We will need a consistent way to generate a JATS file from a publication's metadata for a few different purposes. Bringing this work into OJS will make it available for other purposes, including JATSParserPlugin, OAI, Portico, and our planned work on full-text JATS editing.

**Comment by jonasraoni** [comment]
I'm not aware of how many publication formats exist out there, but looks like JATS is a de-facto standard... Anyway, given the amount of dependencies, it makes sense to have it integrated =]

**Comment by asmecher** [comment]
> I think we should bring this into the core application...

I'd like to have a quick conversation during e.g. a dev call about the best starting point for this -- we're generating JATS both in the Portico plugin and in the JATS Template plugin. I believe the JATS Template plugin has received more finessing as part of Coalition Publica, so IMO it makes sense to start from that, but a comparison against Portico to look for discrepancies woudn't be a bad idea. The metadata plugin toolset and filters are the current best place for it to live (distributed with OJS, even though it's a plugin), not to say that other options aren't possible.
